scripts/o3de.py

```diff
@@ -35,7 +35,7 @@ def add_args(parser: argparse.ArgumentParser) -> None:
     sys.path.insert(0, str(o3de_package_dir))
```

Comment:
with the python based approach, what is the benefit to using o3de.py as the entry point to calling this? Why not just skip invoking o3de.py and call what that calls, directly?

Reply:
We had a discussion on this: The grouping kind of matches other uses of o3de.py, it can help with discoverability.

It also helps with parameter management: o3de.py is setup to be aware of things like engine folder. We may be able to automatically populate properties for the project exporting scripts from o3de.py easier than elsewhere.